neuralactivitycubic/view.py

```diff
@@ -131,6 +132,246 @@ def update_progress_bar(self, progress_in_percent: float) -> None:
         self.progress_bar.value = progress_in_percent
 
 # %% ../nbs/01_view.ipynb 7
+class NWBMetadataPanel:
+
+    @dataclass
+    class NWBMetadata:
+        experiment_description: w.Text = None
+        experimenter: w.Text = None
+        lab: w.Text = None
+        institution: w.Text = None
+        notes: w.Textarea = None
+        model_number: w.Text = None
+        manufacturer: w.Text = None
+        description: w.Textarea = None
+        location: w.Text = None
+        calcium_indicator: w.Text = None
+        excitation_wavelength: w.BoundedFloatText = None
+        emission_wavelength: w.BoundedFloatText = None
+        imaging_description: w.Textarea = None
+
+        @property
+        def current_metadata(self) -> dict[str, Any]:
+            return {
+                'experiment_description': self.experiment_description.value,
+                'experimenter': self.experimenter.value,
+                'lab': self.lab.value,
+                'institution': self.institution.value,
+                'notes': self.notes.value,
+                'model_number': self.model_number.value,
+                'manufacturer': self.manufacturer.value,
+                'description': self.description.value,
+                'location': self.location.value,
+                'calcium_indicator': self.calcium_indicator.value,
+                'excitation_wavelength': self.excitation_wavelength.value,
+                'emission_wavelength': self.emission_wavelength.value,
+                'imaging_description': self.imaging_description.value,
+            }
+
+    def __init__(self) -> None:
+        self.metadata = self.NWBMetadata()
+        self.metadata.experiment_description = w.Text()
+        self.metadata.experimenter = w.Text()
+        self.metadata.lab = w.Text()
+        self.metadata.institution = w.Text()
+        self.metadata.notes = w.Textarea()
+        self.metadata.model_number = w.Text()
+        self.metadata.manufacturer = w.Text()
+        self.metadata.description = w.Textarea()
+        self.metadata.location = w.Text()
+        self.metadata.calcium_indicator = w.Text()
+        self.metadata.excitation_wavelength = w.BoundedFloatText()
```

Comment:
Seems that by default the max value is set to 100.0

Given that typically excitation & emission wavelengths for fluorescence microscopy range between 350 - 1300 nm (for 2-photon applications), that´s clearly not sufficient. 

Let´s change to min = 0.0 [won´t ever be used, but it´s a good default value to indicate that this wasn´t set by the user] and max = 1400.00